queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT * FROM animals WHERE date_of_birth BETWEEN  '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE escape_attempts < 3;
```

Comment:
Missing condition for neutered animals

- [ ] Kindly update query

Reply:
![select](https://user-images.githubusercontent.com/75126481/134151276-bac82056-cfbd-4ff3-b7d4-4243d7a77f46.png)

I think that I put the condition on line 8 please double check

Reply:
- [ ] As requested by the previous Code Reviewer add a query that lists the name of all animals that are neutered and have less than 3 escape attempts. Currently, on line 8, the query is checking for only neutered animals. Please make the changes on line 5.